src/main/java/org/peergos/util/TraceLogger.java

```diff
@@ -40,13 +45,31 @@ public void setIdentity(PeerId peerId) {
         currentNodeId = nodeIdMap.get(peerId.toString());
     }
 
+    /**
+     * Generates a random Hex string of length traceIdSizeInBytes * 2
+     * 
+     * @return traceId The randomly generated traceId.
+     */
+    public String generateTraceId(int traceIdSizeInBytes) {
```

Comment:
Consider using TRACE_ID_SIZE_IN_BYTES directly instead of taking traceIdSizeInBytes as an input.

Reply:
Nice suggestion. Updated.